docs/HyperSync/api-tokens.mdx

```diff
@@ -78,6 +80,13 @@ let client = Client::new(ClientConfig {
   </TabItem>
 </Tabs>
 
+## Understanding Usage
+
+To understand your current month's usage, visit [https://envio.dev/app/api-tokens](https://envio.dev/app/api-tokens). Usage is composed of two main components:
+
+- **Number of Requests**: The total count of API requests made.
+- **Credits**: A calculation based on the endpoint used and the size of the query.
```

Comment:
Right now this calculation is too opaque and scary for a user IMO.

Reply:
What do you suggest? You can see it on the usage dashboard. We can create a separate doc on the calculation if you like but it's probably TMI for a user and subject to change during the course of this month.

Reply:
I think we still haven't even decided on the calc tbh as I know it varies a lot per use case. Maybe just to say the calc takes into account, data bandwidth, disk read etc and is a best representation of usage, we happy to expand upon detail on request?

I know if I was a user and a pricing calc was completley opaque it would be quite nerve wracking so I think we do need to share in more detail somewhere at somestage

Reply:
Cool I'll add that 👍🏼 sounds good.